statesync/stateprovider.go

```diff
@@ -96,6 +97,18 @@ func NewLightClientStateProvider(
 	}, nil
 }
 
+func uniqServers(servers []string) []string {
```

Comment:
Why not unit test this function?

Reply:
It's a private function.

Reply:
I think that it is better to test whether duplication is popping using this function.

Reply:
I understand that the scenario test can be tested in the same way as with one server or empty.